help.go

```diff
@@ -146,8 +146,18 @@ func printHelpCommand(command string) {
 					enum = parameter.Value.Schema.Value.Enum
 				}
 
+				// Convert type to string representation

```

Comment:
nit: type conversion could be a separate function, which takes the first slice element